aplib.net-demo/Assets/Scripts/InputManager.cs

```diff
@@ -59,14 +61,21 @@ private void Start()
         _playerActions.Jump.canceled += _ => _playerMovement.OnJumpUp();
         _playerActions.Respawn.performed += _ => _playerRespawn.Respawn();
         _playerActions.UseItem.performed += _ => _inventory.ActivateItem();
-        _playerActions.NextItem.performed += _ => equipmentInventory.MoveNext();
+        _playerActions.NextItem.performed += _ =>
```

Comment:
I think the scrolling bug comes from this action. You can fix it by deleting the weapon scrolling option in the weapon switch input action